crates/matrix-sdk-common/src/lib.rs

```diff
@@ -28,6 +28,9 @@ pub mod store_locks;
 pub mod timeout;
 pub mod tracing_timer;
 
+#[cfg(target_arch = "wasm32")]
+pub mod wasm_tracing;
```

Comment:
This is not for Wasm. Wasm is a compilation target. This tracing is designed for a JavaScript host running a Wasm module. That's why `matrix-sdk-crypto-js` wasn't named `matrix-sdk-crypto-wasm` initially. Wasm is host agnostic, but this module uses a JS ABI, so it implies a JS host.

Can we rename this module to something like `js_tracing` maybe?

Reply:
I think the point of calling it `wasm` is that wasm is *usually* used in a JavaScript host, whereas there is more than one way to run stuff in a Javascript host (in particular: native plugins in a node interpreter), and this particular implementation relies on wasm.

In other words, in common speech, wasm implies JS, whereas JS does not imply wasm.

Nevertheless I take your point and have renamed accordingly.